pkg/shared/oddmail/go_simple_mail_service_config.go

```diff
@@ -23,6 +23,7 @@ type GoSimpleMailServiceConfig struct {
 	SendTimeout    time.Duration
 	KeepAlive      bool
 	TLSConfig      *tls.Config
+	DevTeamEmail   string
```

Comment:
Would it make more sense to call this `MINTTeamEmail`?

Reply:
Originally I had named it MINTTeamEmail but as this is in the shared package I thought it more appropriate to name it in an agnostic fashion so the package may be more readily shared outside our team at a later date.

Reply:
I find DevTeamEmail confusing as well. Dev team sounds like it is going to the developers of the application, eg us, instead of to but it is going to the business owner team, not the development team. 

Is there any reason that this needs to live in the simple mail service config? This really feels like an application config, and could be set in the appconfig package instead of the shared service config. Thoughts?